bridge-fungible-token/src/utils.sw

```diff
@@ -92,18 +122,23 @@ pub fn adjust_deposit_decimals(msg_val: b256) -> Result<u64, BridgeFungibleToken
         if decimal_diff > 19u8 {
```

Comment:
This check is now covered by the `shift_decimals_right` function

Reply:
I will check again, but initial tests where I removed this check and set decimals to be >= 19 showed this to be still needed.

Reply:
I meant this check is no longer needed. The `shift_decimals_right` function can handle any positive `decimal_diff`